### DISCUSSION TITLE
How to install STT from source?

### DISCUSSION THREAD

**Discussion by Gedeon-m-gedus** [coqui-ai/STT#1833] [Q&A]
ERROR: Could not find a version that satisfies the requirement ds_ctcdecoder==training/coqui_stt_training/VERSION (from coqui-stt-training===training-coqui-stt-training-VERSION) (from versions: 0.9.2, 0.9.3, 0.10.0a3)
ERROR: No matching distribution found for ds_ctcdecoder==training/coqui_stt_training/VERSION (from coqui-stt-training===training-coqui-stt-training-VERSION)

**Comment by reuben** [comment]
Please provide more details on your system. The version should be in a semver format, not a path. That seems to indicate your system is not handling symlinks correctly when cloning. Are you on Windows maybe?

**Reply by Gedeon-m-gedus** [reply]
Yeah, I am on Windows,

I just used Pypi instead and everything worked perfectly, `python3 -m pip install stt`

Thanks

**Reply by reuben** [reply]
Ah, I got confused, you want inference. In that case `pip install stt` is the way to go. Cloning this repo and just doing `pip install .` from the root will install the *training* code, which is not supported on Windows.
